chia/wallet/wallet_transaction_store.py

```diff
@@ -278,10 +307,18 @@ async def get_transactions_between(
         else:
             query_str = SortKey[sort_key].ascending()
 
+        if type_filter is None or len(type_filter.values) == 0:
```

Comment:
```suggestion
        if type_filter is None:
```

Adding the test for the empty `values` array showed that at least in my opinion this is wrong to skip this for empty lists because you then have the same behaviour for adding all types in there which seems not so intuitive, i think? Needs test adjustment also of course if you change it.